backend/routes/system_routes.py

```
@@ -0,0 +1,340 @@
+            age_str = f"{int(age_s)}s ago" if age_s < 120 else f"{int(age_s/60)}m ago"
+        else:
+            age_str = "never"
+
+        events = int(events_today) if events_today else 0
+        return ServiceHealth(
+            status="polling",
+            last_poll=age_str,
+            details=f"events today: {events:,}",
+            paused=paused,
+        )
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)